airlock/business_logic.py

```diff
@@ -189,6 +189,21 @@ def from_dict(cls, attrs) -> Self:
             reviews=[FileReview.from_dict(value) for value in attrs.get("reviews", ())],
         )
 
+    def approved_for_release(self):
+        """
+        A file is approved for release if it has been approved by two reviewers
+        """
+        return (
+            len(
+                [
+                    review
+                    for review in self.reviews
+                    if review.status == FileReviewStatus.APPROVED
```

Comment:
I think it should probably also have no REJECTED reviews?

I guess this would only happen when three output-checker have reviewed the file?

So maybe its a corner case we don't need to worry about

Reply:
I considered that, but my understanding is that it has to have 2 approvals, not that is has to have consensus among all output-checkers. Worth asking Louis, I guess